index.d.ts

```diff
@@ -175,7 +175,8 @@ declare namespace Eris {
   // Application Commands
   interface ApplicationCommand<T extends ApplicationCommandTypes = ApplicationCommandTypes> {
     application_id: string;
-    defaultPermission?: boolean;
+    defaultMemberPermissions?: string;
```

Comment:
Should be able to pass BigInt/Number/String/Permission to be consistent with other permission related methods elsewhere